crates/containerd-shim-wasm/src/sandbox/shim/local.rs

```diff
@@ -326,41 +328,60 @@ impl<T: Instance + Send + Sync, E: EventSender> Local<T, E> {
 }
 
 impl<T: Instance + Sync + Send, E: EventSender> Task for Local<T, E> {
-    #[cfg_attr(feature = "tracing", tracing::instrument(parent = tracing::Span::current(), skip_all, level = "Info"))]
-    fn create(&self, _: &TtrpcContext, req: CreateTaskRequest) -> TtrpcResult<CreateTaskResponse> {
+    #[cfg_attr(feature = "tracing", tracing::instrument(skip_all, level = "Info"))]
+    fn create(
+        &self,
+        ctx: &TtrpcContext,
+        req: CreateTaskRequest,
+    ) -> TtrpcResult<CreateTaskResponse> {
         debug!("create: {:?}", req);
+
+        #[cfg(feature = "opentelemetry")]
+        tracing::Span::current().set_parent(extract_context(&ctx.metadata));
+
         Ok(self.task_create(req)?)
     }
 
     #[cfg_attr(feature = "tracing", tracing::instrument(parent = tracing::Span::current(), skip_all, level = "Info"))]
```

Comment:
Does it need `parent=..` here?
Same for the other methods.

Reply:
what do you mean by `parent=...`?

Reply:
The `parent = tracing::Span::current()` inside the `tracing::instrument` macro.

Reply:
okay you got a point. I will clean them up

Reply:
on a new PR though ^^